conjure.schema.json

```
@@ -503,6 +516,18 @@
         "query"
       ]
     },
+    "EndpointError": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/TypeName"
+        },
+        "docs": {
+          "$ref": "#/definitions/DocString"
+        }
+      }
```

I think we want to describe the `error` field as required when this format is used (see FieldDefinition as an example)
```suggestion
      },
      "required": [
         "error"
      ]
```

---

Makes sense, updated.